### DISCUSSION TITLE
Infinite Scrolling on the Frontpage

### DISCUSSION THREAD

**Discussion by marioortizmanero** [adityatelange/hugo-PaperMod#779] [Ideas]
Although PaperMod attempts to remain as simple as possible (which I love), I feel like infinite scrolling would actually improve the user experience. It's a bit annoying to have to click next/previous page to browse the articles, in my opinion. But this is of course very subjective, so let's see what everyone thinks about it first. I surprisingly haven't found any prior discussion about this.

If most of us agree on this I can open an issue.

**Comment by bikeophilipp** [comment]
Unlimited scrolling works for me if I delete in the config.yal the part with `paginate: 5`. 

I change from this: 

```
baseURL: "https://examplesite.com/"
title: ExampleSite
paginate: 5
theme: PaperMod
```

to this: 
```
baseURL: "https://examplesite.com/"
title: ExampleSite
theme: PaperMod
```

At least for my 8 posts so far it worked. Do not know how it behaves with more.

**Reply by marioortizmanero** [reply]
Thanks for the tip! But is that actual infinite scrolling? I only have a few articles, too. But I imagine that when there are a lot of them it's going to load all of them at once, which is bad.

**Comment by bikeophilipp** [comment]
You are right. This is not intitine scrolling in the typical sense with reload of additional posts once you hit the bottom.
Would also appreviate an Infinite Scolling.
